### ISSUE TITLE
pushError loses context data

### ISSUE THREAD

**Issue by codecapitano** [grafana/faro-web-sdk#486]
## Description

Context data attached to a error via the context object gets lost in the backend when using Grafana Agent.

```ts
aro.api.pushError(error, {
  type: 'network',
  context: {
      message: 'React error boundary',
      componentStackTrace: {...}
    },
});
```

The issue was reported on the Faro community Slack.
I tested it with the Faro Receiver in cloud and the Grafana Agent.
Faro receiver works well, but when using the Agent the context attributes do not appear in the Loki log.

## Steps to reproduce
I started the Demo app which uses the Grafana Agent and sent a custom error with context attributes attached.
The context attributes did not show up in the error log.

Then I started the fe-o11y plugin, which is using the Faro receiver, locally and sent a custom error with context attributes attached as well.
The context attributes showed up in the error log as excpected.

## Expected behavior
Context attributes should appear in the Loki log.

## Actual behavior

context attributes do not appear in the Loki log.

## Environment

- **SDK version: 1.3.8**
- **SDK instrumentations:**
- **Device type:**
- **Device name:**
- **OS:**
- **Browser:**

**Comment by sebastianbaron** [comment]
Having this exact same issue, any updates / fixes in sight?

**Comment by andreieftimie** [comment]
Anyone looking at this?

We think we are running into the same issue.

**Comment by codecapitano** [comment]
Found the issue: https://github.com/grafana/alloy/pull/1413

**Comment by codecapitano** [comment]
The fix will be release with Alloy v1.4

**Comment by codecapitano** [comment]
Alloy v1.4 i already released. Closing this issue
